src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-tabs/smb-tabs.component.ts

```diff
@@ -15,18 +17,18 @@ enum TABS {
 })
 export class SmbTabsComponent implements OnInit {
   selectedTab: TABS;
-  activeTab: TABS = TABS.clusters;
+  activeTab: TABS = TABS.cluster;
 
   constructor(private router: Router) {}
 
   ngOnInit(): void {
     const currentPath = this.router.url;
-    this.activeTab = Object.values(TABS).find((tab) => currentPath.includes(tab)) || TABS.clusters;
+    this.activeTab = Object.values(TABS).find((tab) => currentPath.includes(tab)) || TABS.cluster;
   }
 
   onSelected(tab: TABS) {
     this.selectedTab = tab;
-    this.router.navigate([`/cephfs/smb/${tab}`]);
+    this.router.navigate([`${SMB_PATH}${tab}`]);
```

Comment:
```suggestion
    this.router.navigate([`${SMB_PATH}/${tab}`]);
```
Its always better to avoid adding in the end  "/" in a constant as it helps to follow on later and also to read code

Reply:
Also, I dont see anywhere in dashboard doing such so it might create confusion later for others, so lets be consistent.

Reply:
Udpdated it to `cephfs/smb`